analytical_engine/core/server/graphscope_service.cc

```diff
@@ -82,7 +69,30 @@ ::grpc::Status GraphScopeService::RunStep(::grpc::ServerContext* context,
       return Status(StatusCode::INTERNAL, error_msgs);
     }
 
-    // Second pass: aggregate result according to the policy
+    // Second pass: aggregate graph def according to the policy
+    switch (policy) {
+    case DispatchResult::AggregatePolicy::kMerge: {
```

Comment:
Better to leave some comments about why the graph_def is special and why need to merge, for others can understand this part more quickly.